src/components/app/App.jsx

```diff
@@ -83,12 +87,12 @@ const queryClient = new QueryClient({
   defaultOptions: {
     queries: {
       retry: defaultQueryClientRetryHandler,
-      // Specifying a longer `staleTime` of 60 seconds means queries will not refetch their data
+      // Specifying a longer `staleTime` of 20 seconds means queries will not refetch their data
       // as often; mitigates making duplicate queries when within the `staleTime` window, instead
       // relying on the cached data until the `staleTime` window has exceeded. This may be modified
       // per-query, as needed, if certain queries expect to be more up-to-date than others. Allows
       // `useQuery` to be used as a state manager.
-      staleTime: 1000 * 60,
+      staleTime: 1000 * 20,
```

Comment:
This is intentional. Opting for more frequent query refetches given the change in loading paradigm.